content/client-server-api/modules/account_data.md

```diff
@@ -28,4 +28,9 @@ These events can also be received in a `/events` response or in the
 
 Servers MUST reject clients from setting account data for event types
 that the server manages. Currently, this only includes
-[m.fully\_read](#mfully_read).
+[`m.fully_read`](#mfully_read) and [`m.push_rules`](#mfully_read).
+This applies to both global and room-specific account data.
+
+Servers must allow clients to read the above event types as normal. Note that
+`m.fully_read` only exists for room-specific account data and `m.push_rules`
+only exists for global account data.
```

Comment:
This is a bit confusing when paired with the line above about how *both* types of account data are banned. I think we can get away with links to the respective modules (as above).

```suggestion
Servers must allow clients to read the above event types as normal.
```